### PR TITLE
Update FI

### PR DESCRIPTION
**Pull request by sherryw94**

### CODE REVIEW

File: public/data/travel/FI.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ sources:
   domestic:
   - "[Domestic travel allowed](https://valtioneuvosto.fi/en/article/-/asset_publisher/10616/muutoksia-koronavirusepidemian-vuoksi-asetettuihin-rajoituksiin-1-kesakuuta)
```
Comment (jannwong) [inline public/data/travel/FI.yml:4]: should be single `'` not `"`

File: public/data/travel/FI.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ sources:
...
     as long as health and safety guidelines are observed."
+  - '[Events with more than 500 attendees will be permitted from August 31, 2020](https://valtioneuvosto.fi/en/information-on-coronavirus/current-restrictions), both indoors and outdoors, providing certain social distancing and hygiene measures are adhered to.'
```
Comment (jannwong) [inline public/data/travel/FI.yml:6]: hyperlink can be shortened

**Review by jannwong — CHANGES_REQUESTED** [review]

File: public/data/travel/FI.yml (hunk 1 above)
Comment (sherryw94) [reply public/data/travel/FI.yml:4]: gab's instructions to not change it because it is the original appearance when i open the file.

**Review by jannwong — APPROVED** [review]

**Pull request merged**